managed/services/dbaas/dbaas_client.go

```diff
@@ -54,6 +55,30 @@ func NewClient(dbaasControllerAPIAddress string) *Client {
 	return c
 }
 
+func (c *Client) initializeInClusterK8sClient() error {
+	client, err := NewK8sInclusterClient()
+	if err != nil {
+		return err
+	}
+	c.k8sClient = client
+	return nil
+}
+func (c *Client) GetKubeConfig() (string, error) {
+
+	if err := c.initializeInClusterK8sClient(); err != nil {
+		return "", err
+	}
+	if err := c.Connect(context.Background()); err != nil {
```

Comment:
@BupycHuk I don't like connecting to dbaas-controller here. It kind of breaks logic of enabling/disabling dbaas feature. Any suggestions here?

Reply:
in the case of disabled dbaas, we should return an empty value by default.